src/lib/operators/import_binary.cpp

```diff
@@ -224,4 +249,75 @@ std::shared_ptr<RunLengthSegment<T>> ImportBinary::_import_run_length_segment(st
   return std::make_shared<RunLengthSegment<T>>(values, null_values, end_positions);
 }
 
+template <typename T>
+std::shared_ptr<FrameOfReferenceSegment<T>> ImportBinary::_import_frame_of_reference_segment(std::ifstream& file,
+                                                                                             ChunkOffset row_count) {
+  const auto attribute_vector_width = _read_value<AttributeVectorWidth>(file);
+  const auto block_count = _read_value<uint32_t>(file);
+  const auto block_minima = pmr_vector<T>(_read_values<T>(file, block_count));
+  const auto size = _read_value<uint32_t>(file);
+  const auto null_values = pmr_vector<bool>(_read_values<bool>(file, size));
+  auto offset_values = _import_offset_value_vector(file, row_count, attribute_vector_width);
```

Comment:
_import_offset_value_vector vs _export_compressed_vector

Reply:
`_import_offset_value_vector()` and `_import_attribute_vector()` cannot be simply merged to one `_import_compressed_vector()` function since the constructor interface of the dictionary segment requires a `shared_ptr<BaseCompressedVector>` whereas the RunLengthSegment requires a `unique_ptr<const BaseCompressedVector>`. This could be unified if the segment constructor interfaces are unified as well.

Reply:
Ok, let's leave it for now